### PR TITLE
Fix after css

### PR DESCRIPTION
**Pull request by MTTParry**
-more Heroku changes
-the api works! Adding a debt will send an email to the person who owes the debt.

### CODE REVIEW

File: server/server.js [hunk 1]
```diff
@@ -216,26 +211,39 @@ app.post("/db/debts", cors(), async (req, res) => {
...
   sgMail.setApiKey(process.env.SENDGRID_API_KEY);
   const msg = {
-    to: "michaela.t.t.parry@gmail.com", // Change to your recipient
+    to: `${emailTo.rows[0].email}`, // Change to your recipient
     from: "col.snake.butler@gmail.com", // Verified sender email
-    subject: "Test email",
-    text: "and easy to do anywhere, even with Node.js",
-    html: "<strong>and easy to do anywhere, even with Node.js</strong>",
+    subject: "You have a new debt to pay",
+    text: `Hello ${emailTo.rows[0].first_name} ${emailTo.rows[0].last_name}, You have a new debt to pay! You owe ${debtFrom.rows[0].first_name} ${debtFrom.rows[0].last_name} $${newDebt.how_much}. ${debtFrom.rows[0].first_name} prefers to be paid ${debtFrom.rows[0].preferred_payment_method}. Feel free to contact ${debtFrom.rows[0].first_name} at ${debtFrom.rows[0].email}. Debts Notes: ${newDebt.debt_notes} Bill Notes: ${billInfo.rows[0].bill_notes} -TabSplitter`,
```
Comment (sfurlani-svmk) [inline server/server.js:236]: My preference is to pull out and name variables:
```javascript
const debtor = emailTo.rows[0]
const bill = billInfo.rows[0]
const creditor = debtFrom.rows[0]
```

and then use:
```javascript
`Hello ${debtor.first_name} ${debtor.last_name}, You have a new debt to pay! 
You owe ${creditor.first_name} ${creditor.last_name} $${newDebt.how_much}. 
${creditor.first_name} prefers to be paid ${creditor.preferred_payment_method}. 
Feel free to contact ${creditor.first_name} at ${creditor.email}. 

Debts Notes: ${newDebt.debt_notes} 
Bill Notes: ${bill.bill_notes}`
```

Keeping it in the array `emailTo.rows[0].xxxxx` signifies that there might be information valuable at `emailTo.rows[1] ... emailTo.rows[n]` when there isn't.

Maybe try this:
```javascript
  // name this what it is - query results
  const debtorQueryResults = await db.query(
    "SELECT * FROM contacts WHERE contact_id=$1", 
    [newDebt.who_owes]
  );
  // name this what you're looking for 
  const debtor = debtorQueryResults.rows[0];
  
  // technically you should have an error check here for each query result
  if (!debtor) { 
    res.status(404).end()
  }
```

File: server/server.js [hunk 2]
```diff
@@ -216,26 +211,39 @@ app.post("/db/debts", cors(), async (req, res) => {
...
+
+    html: `Hello ${emailTo.rows[0].first_name} ${emailTo.rows[0].last_name}, <br/>You have a new debt to pay! <br/> You owe ${debtFrom.rows[0].first_name} ${debtFrom.rows[0].last_name} <b>$${newDebt.how_much}</b>. ${debtFrom.rows[0].first_name} prefers to be paid via <b>${debtFrom.rows[0].preferred_payment_method}</b>. <br/>Feel free to contact ${debtFrom.rows[0].first_name} at ${debtFrom.rows[0].email}.<br/> Debts Notes: ${newDebt.debt_notes}<br/>Bill Notes: ${billInfo.rows[0].bill_notes}<br/><br/>-TabSplitter`,
```
Comment (sfurlani-svmk) [inline server/server.js:238]: Also, you can put this in it's own function at the bottom of the file, and parameterize the HTML
```javascript
const emailBody = (debtor, creditor, debt, bill, isHTML) => {
  const n = isHTML ? "<br/>" : "\r\n"; // line break in HTML, newline in plain-text
  const bs = isHTML ? "<b>" : "" // bold start tag
  const be = isHTML ? "</b>" : "" // bold end tag
  
  return `Hello ${debtor.first_name} ${debtor.last_name}, ${n}You have a new debt to pay!${n} You owe ${creditor.first_name} ${creditor.last_name} ${n}${bs}$${debt.how_much}.${be} ${creditor.first_name} prefers to be paid via ${bs}${creditor.preferred_payment_method}${be}. ${n}Feel free to contact ${creditor.first_name} at ${creditor.email}.  ${n}${n}Debts Notes: ${debt.debt_notes} ${n}Bill Notes: ${bill.bill_notes}  ${n}${n}-TabSplitter`
}
```

And then just call it like:
```javascript
  const msg = {
    to: `${debtor.email}`, // Change to your recipient
    from: "col.snake.butler@gmail.com", // Verified sender email
    subject: "You have a new debt to pay",
    text: emailBody(debtor, creditor, newDebt, bill, false),
    html: emailBody(debtor, creditor, newDebt, bill, true)
  };
```

**Review by sfurlani-svmk — APPROVED** [review]

File: server/server.js (hunk 2 above)
Comment (sfurlani-svmk) [reply server/server.js:238]: The `'\r\n` is `Carriage Return Line Feed (CRLF)` format to add newline breaks in the MIME plain text format. it's a windows thing, linux just uses `\n` but the terms come from typewriters (where the line would actually feed, and the carriage would actually return).

File: server/server.js (hunk 1 above)
Comment (MTTParry) [reply server/server.js:236]: Done.
`debtor`, `billInfo`, `payeeInfo`, replace all of the `something.rows[0]`, and added the error checks.

**Pull request merged**